src/wp-content/themes/twentythirteen/functions.php

```diff
@@ -828,3 +828,25 @@ function wp_body_open() {
 		do_action( 'wp_body_open' );
 	}
 endif;
+
+/**
+ * Register Custom Block Styles
+ */
+if ( function_exists( 'register_block_style' ) ) {
+	function twentythirteen_register_block_styles() {
+
+		/**
+		 * Register block style
+		 */
+		register_block_style(
+			'core/button',
+			array(
+				'name'         => 'no-shadow',
+				'label'        => 'No Shadow',
```

Comment:
Should this be translated?

Reply:
It should, thanks for catching that. Should be updated in the latest commit.

Reply:
🚢